test/test_podman_baseline.sh

```diff
@@ -213,20 +219,23 @@ fi
 ########
 # Expected to fail
 ########
-podman $PODMANBASE run --security-opt label=disable alpine sh -c 'touch file.txt && dd if=/dev/zero of=file.txt count=1048577 bs=1'
-rc=$?
-if [ $rc != 0 ];
-then
-    echo "Overlay test outside limits passed"
-else
-    echo "Overlay test outside limits failed"
+
+if [ "$showerror" -ne 1 ]; then
+    podman $PODMANBASE run --security-opt label=disable alpine sh -c 'touch file.txt && dd if=/dev/zero of=file.txt count=1048577 bs=1'
+    rc=$?
+    if [ $rc != 0 ];
+    then
+        echo "Overlay test outside limits passed"
+    else
+        echo "Overlay test outside limits failed"
+    fi
 fi
 
 ########
 # Clean up Podman
 ########
 podman rm --all
-podman rmi --all
+#podman rmi --all
```

Comment:
Why are you not removing the images here?

Reply:
There was a podman bug where if no image existed and you ran that command, it would return a non-zero exit code which when combined with -e was problematic.  I fixed that upstream so, I uncommented this now too.